datalad/support/gitrepo.py

```diff
@@ -2930,6 +2930,8 @@ def get_content_info(self, paths=None, ref=None, untracked='all'):
                     # symlink-nature is a technicality that is dependent
                     # on the particular mode annex is in
                     inf['type'] = 'file'
+                if ref and inf['type'] == 'file':
+                    inf['bytesize'] = int(props.group(4))
```

Comment:
Why not to use named groups in regular expression itself?

Reply:
Good point! RF'ed. Looks much nicer in code, indeed. Thx.